File: Examples/example_2/ex2.cs
using System.Web.Script.Serialization;

public class MyClass
{
    public string ClassName { get; set; }
    public string MethodName { get; set; }
    public string[] Parameters { get; set; }
}

// ...

public void HandleRequest(HttpContext context)
{
    var serializer = new JavaScriptSerializer();
    var json = context.Request.Form["config"];
    var config = serializer.Deserialize<MyClass>(json);

    // Use config object in application logic
    // ...
}

File: Examples/example_2/ex2.java
import com.fasterxml.jackson.databind.ObjectMapper;

public class MyClass {
    private String className;
    private String methodName;
    private String[] parameters;

    // Getters and setters omitted for brevity

    @Override
    public String toString() {
        return "MyClass{" +
                "className='" + className + '\'' +
                ", methodName='" + methodName + '\'' +
                ", parameters=" + Arrays.toString(parameters) +
                '}';
    }
}

// ...

public void handleRequest(HttpServletRequest request, HttpServletResponse response) {
    try {
        String json = request.getParameter("config");
        ObjectMapper objectMapper = new ObjectMapper();
        MyClass config = objectMapper.readValue(json, MyClass.class);

        // Use config object in application logic
        // ...
    } catch (IOException e) {
        // Handle exception
        // ...
    }
}
